datacake-cluster/src/storage.rs

```diff
@@ -736,6 +761,13 @@ pub mod test_suite {
             )
             .await
             .expect("Merk documents as tombstones");
+        // NOTE: i manually added this call as the test seems to expect the entry in the keyspace
+        //       to have been removed
+        storage
+            .remove_tombstones(KEYSPACE, [1_u64, 2_u64, 4_u64].into_iter())
+            .await
+            .unwrap();
+
```

Comment:
Mentioned it below, but you can't do this, tombstones can potentially live forever theoretically, the data can be removed but the metadata must stay, and any `get_x` requests afterwards should act as if the data is removed.